install/update.php

```diff
@@ -245,6 +245,9 @@ function showSecurityKeyCheckForm()
             echo "<div class='text-center'>";
             doUpdateDb();
             echo "</div>";
+            echo '<div class="text-center fw-bold">';
+            echo '<p>' . __('It is highly recommended to run the "php bin/console glpi:database:check_schema_integrity" command to verify the database schema integrity.') . '</p>';
+            echo "</div>";
```

Comment:
Unless I am wrong, this is redundant with output of `Update::checkSchemaIntegrity()`